assets/javascripts/discourse/templates/components/chat-message-info.hbs

```diff
@@ -0,0 +1,41 @@
+<div class="chat-message-info">
+  {{#if @message.chat_webhook_event}}
+    {{#if @message.chat_webhook_event.username}}
+      <span class="chat-message-info__username {{this.usernameClasses}}">
+        {{@message.chat_webhook_event.username}}
+      </span>
+    {{/if}}
+
+    <span class="chat-message-info__bot-indicator">
+      {{i18n "chat.bot"}}
+    </span>
+  {{else}}
+    <span
+      role="button"
+      class="chat-message-info__username {{this.usernameClasses}} clickable"
+      data-user-card={{@message.user.username}}
+    >
+      <span class="chat-message-info__username__name">{{this.name}}</span>
+
+      <span class="chat-message-info__user-avatar-flair">
+        {{user-avatar-flair user=@message.user}}
+      </span>
+    </span>
+  {{/if}}
+
+  <span class="chat-message-info__date">
+    {{format-chat-date @message @details}}
+  </span>
+
+  {{#if (or @message.reviewable_id (eq @message.user_flag_status 0))}}
```

Comment:
I find this a little hard to follow, is it `if the message has a reviewable id AND the user flag status equals 0` or is it `if the message has a reviewable id OR the user flag status equals 0`?